pkg/deploy/deploy.go

```diff
@@ -185,8 +185,8 @@ func (d *GeminiDeployer) prepareForUpload() error {
 		return util.UnexpectedNil
 	}
 	for ip, r := range d.remotes {
-		binPath := r.UpDataPath + util.Remote_bin_rel_path
-		etcPath := r.UpDataPath + util.Remote_etc_rel_path
+		binPath := r.UpDataPath + d.version + util.Remote_bin_rel_path
```

Comment:
The file splicing method uses `filepath.Join`, which can be cross-platform.

Please check other similar codes

Reply:
I have fixed all similar codes.